services/storage/src/simcore_service_storage/api/rpc/_data_export.py

```diff
@@ -60,6 +61,10 @@ async def start_data_export(
         task_uuid = await get_celery_client(app).send_task(
             "export_data",
             task_context=job_id_data.model_dump(),
+            task_metadata=TaskMetadata(
+                ephemeral=False,
```

Comment:
question: does it make sense to define this task as ephemeral from storage?
question2: why do we even this ephemeral thing? Just wondering, as client I know what are the tasks I am interested to know about depending on the context I guess? How will this affect the frontend experience? does that mean that if I duplicate 50 projects during 1 hour, everytime I F5 it will list many tasks? or is this independent?

Reply:
??